assets/js/src/form_editor/blocks/custom_checkbox/custom_field_settings.jsx

```diff
@@ -53,6 +53,7 @@ const CustomFieldSettings = ({
         />
         <input
           value={localCheckboxLabel}
+          data-automation-id="custom_field_value_settings_value"
```

Comment:
The word _value_ is twice in this ID. Do you think that we could change it?

Reply:
Yes, you're right, my fault.

Reply:
Yes, I changed it to be more concise.